frontend/plugins/pos_ui/src/modules/create-pos/components/sync/sync.tsx

```
@@ -4,8 +4,16 @@ import { useSearchParams } from 'react-router-dom';
 import { syncCardSettingsAtom } from '../../states/posCategory';
+import { IPosDetail } from '~/modules/pos-detail/types/IPos';
+import { SelectBranch, SelectMember } from 'ui-modules';
 
-export default function SyncCardForm() {
+interface SyncCardFormProps {
+  posDetail?: IPosDetail;
+  isReadOnly?: boolean;
+}
+
+export default function SyncCardForm({ 
+}: SyncCardFormProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)